static_report/src/components/ComparisonReport/CRColumnDetailsRow.tsx

```diff
@@ -0,0 +1,85 @@
+import { Box, Flex, Text } from '@chakra-ui/react';
+import { Link, useLocation } from 'wouter';
+import { ColumnSchema } from '../../sdlc/single-report-schema';
+import { ZColSchema, zReport } from '../../types';
+import { checkColumnCategorical } from '../../utils/transformers';
+import { ColumnTypeHeader } from '../shared/Columns/ColumnTypeHeader';
+import { NO_VALUE } from '../shared/Columns/constants';
+import { MetricsInfo } from '../shared/Columns/ColumnMetrics/MetricsInfo';
+import { CRGeneralStats } from '../shared/Columns/ColumnMetrics/CRGeneralStats';
+import { CRSummaryStats } from '../shared/Columns/ColumnMetrics/CRSummaryStats';
+
+type Props = {
+  baseColumn?: ColumnSchema;
+  targetColumn?: ColumnSchema;
+};
+export const CRColumnDetailsRow = ({ baseColumn, targetColumn }: Props) => {
+  const fallback = baseColumn || targetColumn;
+  const isCategorical = checkColumnCategorical(baseColumn);
+  const [currentLocation] = useLocation();
+
+  zReport(ZColSchema.safeParse(baseColumn));
+  zReport(ZColSchema.safeParse(targetColumn));
+
+  return (
+    <Flex
+      direction="column"
+      minH="250px"
+      border={'1px solid darkgray'}
+      rounded={'2xl'}
+    >
+      {fallback && (
+        <ColumnTypeHeader
+          columnDatum={fallback}
+          bg={'blue.800'}
+          color={'white'}
+        />
+      )}
+      <Box m={4}>
+        <MetricsInfo
+          name=""
```

Comment:
Maybe give a default value `""` in `MetricsInfo`?

Reply:
I do want to keep it flexible, so that if developer wants to only have 
`<first-slot> | <second-slot>` instead of
`<name> | <first-slot> | <second-slot>`, it is possible. 

Else, the component would need to check if `name.length === 0` to render `<name>` at all